src/ConstructiveSolidGeometry/VolumePrimitives/VolumePrimitives.jl

```diff
@@ -1,5 +1,25 @@
-include("Cone.jl")
-include("Box.jl")
-include("RegularPrism.jl")
-include("Sphere.jl")
-include("Torus.jl")
+ClosedPrimitive(p::VP) where {VP <:AbstractVolumePrimitive} = VP(p, COT = ClosedPrimitive, origin = p.origin, rotation = p.rotation)
+OpenPrimitive(p::VP) where {VP <: AbstractVolumePrimitive} = VP(p, COT = OpenPrimitive, origin = p.origin, rotation = p.rotation)
```

Comment:
This one does not care about `scaling`?

Reply:
The current scale is in the fields of `p`. Here we only want to change the parametric type of the primitive but not its fields.
Hmm, one could change the constructor of  `Box(b::Box)` such that the defaults for the keywords `origin` and `rotation`
are the `origin` and `rotation` of `b`. I guess that would make sense as well. :thinking: 
I will have a look and test if it creates issues at other stages.

Reply:
I was confused that one of the transformations was `VP(p, scaling = ...)`. I was expecting that this is needed everywhere..

Reply:
I changed the constructor for `Box(b::Box)` and removed also origin and rotation keywords here.